app/controllers/playlist_items_controller.rb

```diff
@@ -62,6 +62,27 @@ def update
     render json: { message: "Item was not updated: #{error.message}" }, status: 500 and return
   end
 
+  def show
+    playlistItem = PlaylistItem.find(params[:id])
```

Comment:
I think we need authorization here to ensure that the requestor `can? :read, playlistItem`.  This would probably be easiest to do by removing the find on line 66 and adding at the top of the class:
`load_and_authorize_resource only: [:show]`
This will find the playlist item and should put it in `@playlist_item` after performing the `can? :read` check.  If the check fails we should handle it returning json instead of the standard redirect with flash message.  For example look at the create method for an example or doing that although I think we'd want two rescues: 
 `CanCan::AccessDenied` (403) and `StandardError` (500).

Reply:
@cjcolvar Ok, sounds good.  Whatever you recommend for `playlist_items_controller.rb`, let's do it.  I can give it a shot, but might need some assistance.

Reply:
Maybe since this is a show action for playlist_items and not just markers, it would be wise to return {markers: [marker, marker, ...]}, instead of just [marker, marker, ...], and parse it as necessary on the other end.

Reply:
Any chance either of you guys can just adjust `playlist_items_controller.rb` however you think is best, then I'll make updates on the JS side of things?   I'm not really sure how to move on Chris's original comment...   Or if you want to pair on it, I can share a screen and you could  instruct me where to place stuff?

Reply:
I'll work on the controller changes this afternoon.

Reply:
Thanks Brian